docs/LoaderLayerInterface.md

```diff
@@ -1746,6 +1749,608 @@ They do not need to implement functions that they do not intercept.
+        </small>
+    </td>
+  </tr>
+  <tr>
+    <td><small><b>LLP_LAYER_4</b></small></td>
+    <td>A layer <b>should</b> export the
+        <i>vkNegotiateLoaderLayerInterfaceVersion</i> entry-point to negotiate
+        an interface version.<br/>
+        A layer using interface 2 or newer <b>must</b> export this function.<br/>
+    </td>
+    <td>The layer will not be loaded.</td>
+    <td>No</td>
+    <td>Yes</td>
+    <td><small>
+        <a href="#layer-version-negotiation">Layer Version Negotiation</a>
+        </small>
+    </td>
+  </tr>
+  <tr>
+    <td><small><b>LLP_LAYER_5</b></small></td>
+    <td>A layer <b>must</b> be able to negotiate a supported version of the
+        loader/layer interface with the loader in accordance with the stated
+        negotiation process.
+    </td>
+    <td>The layer will not be loaded.</td>
+    <td>No</td>
+    <td>Yes</td>
+    <td><small>
+        <a href="#loader-and-layer-interface-negotiation">
+        Interface Negotiation</a></small>
+    </td>
+  </tr>
+  <tr>
+    <td><small><b>LLP_LAYER_6</b></small></td>
+    <td>A layer <b>must</b> have a valid JSON manifest file for the
+        loader to process that ends with the ".json" suffix.
+        It is recommended validating the layer manifest file against
+        <a href="https://github.com/LunarG/VulkanTools/blob/master/vkconfig_core/layers/layers_schema.json">
+        the layer schema</a> prior to publication.</br>
+        The <b>only</b> exception is on Android which determines layer
+        functionality through the introspection functions defined in
+        <a href="#layer-library-api-version-0">Layer Library API Version 0</a>
+        section and in the
+        <a href="#layer-manifest-file-format">Layer Manifest File Format</a>
+        table.
+    </td>
+    <td>The layer will not be loaded.</td>
+    <td>No</td>
+    <td>Yes</td>
+    <td><small>
+        <a href="#layer-manifest-file-usage">Manifest File Usage</a></small>
+    </td>
+  </tr>
+  <tr>
+    <td><small><b>LLP_LAYER_7</b></small></td>
+    <td>If a layer is a Meta-layer, each component layer in its manifest file
+        <b>must</b> be present on the system.
+    </td>
+    <td>The layer will not be loaded.</td>
+    <td>No</td>
+    <td>Yes</td>
+    <td><small>
+        <a href="#meta-layers">Meta-Layers</a></small>
+    </td>
+  </tr>
+  <tr>
+    <td><small><b>LLP_LAYER_8</b></small></td>
+    <td>If a layer is a Meta-layer, each component layer in its manifest file
+        <b>must</b> report the same Vulkan API major and minor version.
+    </td>
+    <td>The layer will not be loaded.</td>
+    <td>No</td>
+    <td>Yes</td>
+    <td><small>
+        <a href="#meta-layers">Meta-Layers</a></small>
+    </td>
+  </tr>
+  <tr>
+    <td><small><b>LLP_LAYER_9</b></small></td>
+    <td>A layer installed as an Implicit Layer <b>must</b> define and support a
+        disable environment variable so it can be disabled globally.
```

Comment:
Whats the difference between "defining" and "defining and supporting" a disable environment? The loader is responsible for checking the disable env-var, not the layer. So all the layer has to do is add the field in the json.

Reply:
And follow it.  Where do you say that they actually must use that variable.  That's the "support" portion.

Reply:
Oh wait.  It's the loader that does this.  I'll clean it up.